src/Options.cc

```diff
@@ -106,6 +106,8 @@ void usage(const char* prog, int code)
 	fprintf(stderr, "    -s|--rulefile <rulefile>        | read rules from given file\n");
 	fprintf(stderr, "    -t|--tracefile <tracefile>      | activate execution tracing\n");
 	fprintf(stderr, "    -u|--usage-issues               | find variable usage issues and exit\n");
```

Comment:
Mostly wondering if the new option (and possibly the old `--usage-issues`) could have another word added so they don't look so related and easy to confuse.
```
--usage-issues
--no-usage-warnings
```
I do not have great suggestions. Those come to mind: `--variable-usage-issues` or `--variable-usage-warnings` and `--no-func-usage-warnings`, or `--no-invoke-usage-warnings`, or `--Wno-invoke-warnings`.

Reply:
I realized that a better name is `--no-unused-warnings`, which also helps with the name conflict.  Latest push includes that